how-it-works/5_governance/sns.card.md

```diff
@@ -10,4 +10,4 @@ coverImage: /img/how-it-works/network-nervous-system-nns.600x300.jpg
 
 An SNS would derive from the [Network Nervous System (NNS)](/how-it-works/network-nervous-system-nns/) , the autonomous tokenized governance system that controls the Internet Computer blockchain in a completely open, permissionless, and decentralized manner.
 
-* [How the Service Nervous System (SNS) Will Bring Tokenized Governance to On-Chain Dapps | Medium](https://medium.com/dfinity/how-the-service-nervous-system-sns-will-bring-tokenized-governance-to-on-chain-dapps-b74fb8364a5c)
\ No newline at end of file
+[Go deeper](how-it-works/sns/)
```

Comment:
produces a 404

Reply:
Seems this is because the file is just being added now. Works locally and linking is consistent with the others